hack/verify-vendor.sh

```diff
@@ -1,12 +1,9 @@
 #!/bin/sh
 
 if [ "$IS_CONTAINER" != "" ]; then
-  if [ ! "$(command -v dep >/dev/null)" ]; then
-    go get -u github.com/golang/dep/cmd/dep
-  fi
-
-  dep check
-  dep ensure
+  export GO111MODULE=on
+  GOPROXY=https://proxy.golang.org go mod vendor
+  go mod verify
```

Comment:
Note: `go mod verify` doesn't actually verify vendor contents (yet, at least).
https://github.com/golang/go/issues/27348

Reply:
is it useful to split `setup/check Go` to separate lib so that hack/build and this can use it?

Reply:
not important though.

Reply:
Go 1.13 defaults GOPROXY to to `https://proxy.golang.org,direct`. Removed again.